packages/@okta/vuepress-site/docs/guides/style-the-widget/style-self-hosted/index.md

```diff
@@ -0,0 +1,241 @@
+```
+
+### Custom buttons
+
+You can add buttons below the **Sign In** button.
+
+```javascript
+var config = {
+  ...
+  customButtons: [
+    {
+      title: 'Click Me 1',
+      className: 'btn-customAuth',
+      click: function() {
+          // clicking on the button navigates to another page
+          window.location.href = 'http://www.example1.com';
+        }
+    },
+    {
+      title: 'Click Me 2',
+      className: 'btn-customAuth',
+      click: function() {
+          // clicking on the button navigates to another page
+          window.location.href = 'http://www.example2.com';
+      }
+    }
+  ],
+  ...
+};
+```
+
+### Links
+
+You can also change the **Help**, **Forgot Password**, and **Unlock** links, including both their text and URLs.
+
+```javascript
+var config = {
+  ...
+  helpLinks: {
+    help: 'https://example.com/help',
+    forgotPassword: 'https://example.com/forgot-password',
+    unlock: 'https://example.com/unlock-account',
+    custom: [
+      {
+        text: 'What is Okta?',
+        href: 'https://example.com/what-is-okta'
+      },
+      {
+        text: 'Example Portal',
+        href: 'https://example.com'
+      }
+    ]
+  },
+  ...
+};
+```
+
+## Modify the CSS
+
+In addition to the parameters in the Widget's `config`, you can also modify the CSS or override the default styles with your own.
+
+### Modify the existing theme
+
+If you want to add on top of the Okta theme, just edit [okta-theme.scss](https://github.com/okta/okta-signin-widget/blob/master/assets/sass/okta-theme.scss) and add any CSS to the bottom of the file.
+
+### CSS customization examples
+
+This section provides examples that you can use to make your own customizations to the widget.
+
+#### Background
+
+```css
+#okta-sign-in.auth-container.main-container {
+    background-color: red;
+}
+
+#okta-sign-in .beacon-blank {
+    background-color: red;
+}
+```
+
+#### Border color
+
+```css
+#okta-sign-in.auth-container.main-container {
+    border-color: red;
+}
+
+#okta-sign-in.auth-container .okta-sign-in-header {
+    border-bottom-color: red;
+}
+```
+
+#### Text color
+
+All text:
+
+```css
+#okta-sign-in * {
+    color: red;
+}
+```
+
+**Sign In** text:
+
+```css
+#okta-sign-in.auth-container .o-form-head {
+    color: red;
+}
+```
+
+Link text:
+
+```css
+#okta-sign-in.auth-container a.link:link {
+    color: red;
+}
+```
+
+#### Widget positioning + width
+
+Width:
+
+```css
+#okta-sign-in {
+    width: 600px;
+}
+```
+
+Position:
+
+```css
+#okta-sign-in {
+    margin: 100px auto 8px;
+}
+```
+
+#### Video Tutorial
+
+For a more in-depth look at styling the widget, you can watch this video:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/Q__ugprsOWo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+#### Live Widget
+
+You can also play with the SASS in real time with the [live widget](https://developer.okta.com/live-widget).
```

Comment:
According to the [website](https://sass-lang.com/), Sass should not all be capitalized.